.rubocop.yml

```diff
@@ -3,47 +3,6 @@ AllCops:
     - 'data/**/*'
     - 'vendor/**/*'
```

Comment:
It's already in the config that comes with Rubocop by default. 

We discovered last week, however, that if you supply your own Exclude list at all, it completely clobbers the built-in one, rather than extending it. So you need to bring back the defaults manually that you want from that. 

In other words, if you don't have any Excludes, it'll automatically skip 'vendor', so I don't think we need it in _our_ defaults. We only need to re-include any time we have any project-specific Excludes.